guides/common/modules/con_disaster-recovery-with-two-active-project-servers.adoc

```diff
@@ -0,0 +1,5 @@
+[id="disaster-recovery-with-two-active-{project-context}-servers"]
+= Disaster recovery with two active {ProjectServer}s
+
+To prepare for disaster recovery, you can configure two {ProjectServer}s and operate each server in a different data center.
+If one of the servers fails, you can re-register all hosts from the failed server to the other server.
```

Comment:
"re-register" - [ISG](https://www.ibm.com/docs/en/ibm-style?topic=grammar-prefixes) says not to hyphenate prefixes unless required for clarity (examples: re-create, re-enable).

In this situation, "re-register" (or "reregister") seems unnecessary. Although the hosts were registered in the past, to the failed server, the user is registering the previously registered hosts to a functioning server. I think this would be clearer: "If one server fails, you can register its hosts to the other server."

Reply:
Thanks! I wasn't aware of the IBM guide entry.

Searching for "re-register" in the docs set did reveal several cases where we refer to the process of repeated registration as "re-registering". Some of them are in the other DR scenarios that have been reviewed and merged before, but others in unrelated modules that have existed before. Based on that, I'll leave the wording as is here. If using a hyphenated prefix in this context is an issue, I'd prefer to tackle it repo-wide rather than in this single PR.